docs/preview/index.md

```diff
@@ -32,12 +32,6 @@ Let's do it! Pressing `[Enter]` selects the option in brackets `[]`.
 !!! warning "Choose Python 3.6"
     This tutorial takes you through the creation of a containerized instance. Therefore, only Python 3.6 is supported, if you wish to use other versions please follow the [development](../develop/index.md) tutorial.
 
-!!! info "Flavour is now an argument"
```

Comment:
I would keep this since last release many people did not deploy, mainly cuz there were mainly backend work and not much visible.